platform/posix/transport/src/openssl_posix.c

```diff
@@ -460,10 +479,10 @@ OpensslStatus_t Openssl_Connect( NetworkContext_t * pNetworkContext,
 {
     SocketStatus_t socketStatus = SOCKETS_SUCCESS;
     OpensslStatus_t returnStatus = OPENSSL_SUCCESS;
-    int64_t verifyPeerCertStatus = X509_V_OK;
     int32_t sslStatus = 0;
     uint8_t sslObjectCreated = 0;
     SSL_CTX * pSslContext = NULL;
+    int32_t verifyPeerCertStatus = X509_V_OK;
```

Comment:
Why was the bit-width reduced for this? It seems to be adding an explicit cast for the return value from `SSL_get_verify_result`...

Reply:
I have looked at the value range and the max value is ~75.  Also, our implementation only uses the information of whether the return value is X509_V_OK(0) or not. A type with the same width of the return type of SSL_get_verify_result is going to be of 64 bits and this was an opportunity we identified that we could optimize.